resources/lib/playerinfo.py

```diff
@@ -198,8 +215,8 @@ def push_position(self, position=0, total=100, event=None):
             asynchronous=True
         )
 
-        # Do not reload container when we are not done playing
-        if event not in ('onPlayBackEnded', 'onPlayBackError', 'onPlayBackStopped'):
+        # Do not reload container when a single episode was played
```

Comment:
Why? I still do not completely understand our strategy here.
Is this because we do a reload elsewhere as well?

*(I had it disabled because with `onThreadExit()` it appeared to work without reload)*

Reply:
No, in this implementation there is only an extra reload after watching multiple episodes because for some reason Kodi doesn't update the last one of multiple watched episodes.
For watching single episodes, I rely on Kodi internal watch status.

It seems Kodi internal watch status is only updated when the play action is initiated from the GUI, so this only works for single episodes.

Reply:
Thanks for clarifying. That information is crucial to have as comments in the code.
Not just because I have a terrible memory, but also because we may not be around :-)

Reply:
I'll extend the comment with this information.